app/src/main/java/de/westnordost/streetcomplete/quests/max_speed/AddMaxSpeed.kt

```diff
@@ -56,6 +56,16 @@ class AddMaxSpeed : OsmFilterQuestType<MaxSpeedAnswer>() {
                 changes.modify("highway", "living_street")
             }
             is ImplicitMaxSpeed -> {
+                if (answer.countryCode == "GB") {
+                    // nsl_restricted only gets added if the road is lit (whether already set or answered by the user in this quest)
+                    if (answer.roadType == "nsl_restricted") {
+                        changes.add("lit", "yes")
```

Comment:
what is happening in case where lit tag is present already? (maybe `addOrModify` should be used or maybe not)

Reply:
Oh yes, I realised that later, but was in the branch for adding national speed limit